README.md

```diff
@@ -72,3 +102,21 @@ ubuntu@build-snapcraft-io:~/work$ npm test
 
 Emacs is confused if 'build.snapcraft.io' is used in the project directory
 name, using 'build-snapcraft-io' restores sanity.
+
+## Updating dependencies
+
+Managing dependencies with npm can be a little .... interesting, especially if
+you're not used to the peculiarities of the nodejs toolchain.
+
+First, you must add or update the dependency in package.json. This file
+contains both a 'dependencies' section (for run-time dependencies) and a
+'devDependencies' section (for build-time dependencies). Edit the appropriate
+section to add/remove/edit the dependencies that you care about. Version
+numbers are specified using a [special syntax](https://docs.npmjs.com/misc/semver#prerelease-identifiers).
+
+Next, run `npm update` to get the new dependencies locally.
+
+Finally, run `./node_modules/shonkwrap/shonkwrap` (I wish I was joking) to
```

Comment:
Not sure if updating dependencies and docs about shronkwrap should be part of this PR, but as they are already here...

Yes, the need for 'shonkwrap' is quite peculiar in npm (it's said to be much better in newer versions of npm, but we are not there yet) and usage of shonkwrap was not really documented anywhere (just silently assumed by devs working on this project). So adding it to dev dependencies is a good point, so it is not necessary as external tool.

Taking it a step further (to avoid calling it directly out from `./node_modules/...`) we could add a npm script to package.json, like:

```json
{
  scripts: {
      "shonkwrap": "shonkwrap"
  }
}
```

npm scripts can read project dependencies, so it could be run as `npm run shonkwrap` instead of accessing from the path.